api/components/image.schema.json

```diff
@@ -12,16 +12,71 @@
             ]
         },
         "path": {
-            "description": "The image path from the resources folder",
+            "description": "The image path from the resources folder or from the network depending on the value of fromNetwork.",
             "type": "string"
         },
+        "fromNetwork": {
+            "description": "Whether the image will be loaded from a network url of from an application's resource path.",
+            "type": "boolean",
+            "default": false
+        },
         "width": {
-            "description": "The image width",
+            "description": "The image width.",
             "type": "number"
         },
         "height": {
-            "description": "The image height",
+            "description": "The image height.",
             "type": "number"
+        },
+        "alignment": {
+            "description": "How to align the image.",
+            "$ref": "defs/alignment.schema.json"
+        },
+        "centerSlice": {
+            "description": "The center slice for a nine-patch image.",
+            "$ref": "defs/rect.schema.json"
+        },
+        "errorBuilder": {
+            "description": "The error callback when the image encounters an error during loading.",
```

Comment:
This is not a "Callback", just a component. 
Also, since this is not a callback, we might want to change the name to something like "errorPlaceholder" ?

Reply:
This is actually a callback on Flutter, we only implemented the return value of the callback which is a component. 
This work is expected to be temporary as we might need to get the parameters of the callback to have more control over the errorBuilder.

Should we still change the name and the type to component ?

Reply:
Yeah but components-api is not flutter components anymore. Since we cannot create an actual callback with Lenra, we might want to change the property accordingly.

IMO the terms "callback" and a "builder" are not correct here since the developper write json that represent a component.

I think we should change the name and description to match the type (component).